### PR TITLE
3rd Small Improvements

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Troll emperors are called God Kings/Queens.
- The Horde Invasion ends in the year 603 now.
- Mentioned the Uninvited Guest as a goblin's evil god and the Everlasting Party as their afterlife (localization sometimes seen in events).
- Added new religious symbol names (localization sometimes seen in events).
- Troll Loa pantheons have the Adorcism tenet.
- Troll Loa pantheons, Agamagganic and Goldrinnism got Blood Feast tenet that combines Human Sacrifice and Cannibalism.
- Goldrinnism has the Natural Primitivism.
- Possessed trait is a Shath'gral and Doombringer virtue.
- The Lunatic trait is a Shath'gral virtue.

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
Play the game, test each feature. Don't forget to use console commands.

- `set_date 602` to set the year to 602.

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/648/files?file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here are the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).

### CODE REVIEW

**Review by arithon — APPROVED** [review]
loc LGTM

**Review by ercarp — COMMENTED** [review]
> * Troll emperors are called God Kings/Queens.
> * Loa faiths have the Adorcism tenet.
> * Possessed trait is a Shath'gral and Doombringer virtue.
> * The Lunatic trait is a Shath'gral virtue.

Confirmed all of these work perfectly in-game and I have nothing bad to say about any of them. I think they're great additions and make sense. 👍

> * The Horde Invasion ends in the year 603 now.

I wasn't able to test this properly as The Horde lost to Magni in 593 and after that they were just defending against seemingly random incursions. I didn't see any more "Great Invasion" wars pop up.

> * Loa faiths, Agamagganic and Goldrinnism got Blood Feast tenet that combines Human Sacrifice and Cannibalism.

Somehow I feel like this tenet would also work _extremely_ well for the **_G'huuni_** faith, seeing as they're kind of a fusion of troll and old god religion. Perhaps Ritual Cannibalism could be replaced with it? Let me know what you think.

From [Wowpedia](https://wowpedia.fandom.com/wiki/Blood_troll):

> Blood trolls are cannibals, the whispers of their god telling them that eating flesh makes them stronger. Females of the Natha'vor tribe cannibalize particularly on the males within the tribe, and call their three most powerful witch doctors "corpse mongers".

Although there is also this part which works against Blood Feast for G'huuni, since it seems like the blood trolls prefer to capture their foes rather than executing them:

> Blood troll poisons are made to paralyze, not kill because they want prisoners alive

> * Mentioned the Uninvited Guest as a goblin's evil god and the Everlasting Party as their afterlife (localization sometimes seen in events).
> 
> * Added new religious symbol names (localization sometimes seen in events).

Localization looks good to me. 👍 📚

> * Goldrinnism has the Natural Primitivism.

I think this fits their faith extremely well, but isn't it supposed to make characters unclothed? Or was that part of the trait disabled on purpose?

<details><summary>Click to expand</summary>

image

</details>

**TLDR**: Everything looks good to me but I wasn't able to test the 603 invasion end date properly and I'm confused as to whether Natural Primitivism is working as intended or not.

**Comment by zumbak04** [comment]
> Somehow I feel like this tenet would also work extremely well for the G'huuni faith

They don't need it. They can sacrifice humans due to the special doctrine.

> I think this fits their faith extremely well, but isn't it supposed to make characters unclothed?

It was disabled.

**Comment by ercarp** [comment]
> They don't need it. They can sacrifice humans due to the special doctrine.

Gotcha.

> It was disabled.

Fair enough. ~~No night elf booba this time I guess. 😢~~

I also tested the new invasion end date using the `set_date` command and I can confirm the invasion ended abruptly around June or July 603. I tested it twice.

**Review by ercarp — APPROVED** [review]

**Comment by ValianBlue** [comment]
> Troll emperors are called God Kings/Queens.

✅ 

> The Horde Invasion ends in the year 603 now.

✅

> Mentioned the Uninvited Guest as a goblin's evil god and the Everlasting Party as their afterlife (localization sometimes seen in events).

✅

> Added new religious symbol names (localization sometimes seen in events).

_Unable to test._

> Loa faiths have the Adorcism tenet.

❌ _Torgi, Ariwi, Sethrali, and Hakkari are listed as Loa faiths but do not have the Adorcism tenet. This may be intended, but I am making a note of it here._

> Loa faiths, Agamagganic and Goldrinnism got Blood Feast tenet that combines Human Sacrifice and Cannibalism.

✅ The Blood Feast tenet works correctly.

❌ _Zan'atali, Sethrali, Ariwi, and Torgi do not have the Blood Feast tenet. Again, I expect this is intentional but I am noting it just to double check._

> Goldrinnism has the Natural Primitivism.

✅

> Possessed trait is a Shath'gral and Doombringer virtue.

✅

<details>
<summary>With so many virtues, they are pushed into two rows not showing any tooltips.</summary>

image
</details>

> The Lunatic trait is a Shath'gral virtue.

✅

**Comment by zumbak04** [comment]
> Torgi, Ariwi, Sethrali, and Hakkari are listed as Loa faiths but do not have the Adorcism tenet. This may be intended, but I am making a note of it here.

It's intended.

> Zan'atali, Sethrali, Ariwi, and Torgi do not have the Blood Feast tenet. Again, I expect this is intentional but I am noting it just to double check.

It's intended. They don't practice cannibalism.

> With so many virtues, they are pushed into two rows not showing any tooltips.

Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/651.

**Review by ValianBlue — APPROVED** [review]
_Thank you for clarifying._

_Issue #651 reviewed and approved. With that, everything is good to go._
